doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+
+Just like in other Python container objects, the contents of an array can be accessed and modified by indexing or slicing the array. Different arrays can share the same data, so changes made on one array might be visible in another. 
+
+Array **attributes** reflect information intrinsic to the array itself. If you need to get, or even set, properties of an array without creating a new array, you can often access an array through its attributes. 
+
+`Read more about array attributes here <https://docs.scipy.org/doc/numpy/reference/arrays.ndarray.html>`_ and learn about `array objects here <https://docs.scipy.org/doc/numpy-1.17.0/reference/arrays.html>`_.
+
+
+How to create a basic array
+---------------------------
+
+
+::
+
+  np.array()
+  np.zeros() 
+  np.ones() 
+  np.empty() 
+  np.arange() 
+  np.linspace()
+  dtype
+
+-----
+
+To create a NumPy array, you can use the function ``np.array()``.
+
+All you need to do to create a simple array is pass a list to it. If you choose to, you can also specify the type of data in your list. `You can find more information about data types here <https://numpy.org/devdocs/user/quickstart.html#arrays-dtypes>`_.
+
+::
+
+    import numpy as np
+
+    a = np.array([1, 2, 3])
+
+You can visualize your array this way:
+
+.. image:: images/np_array.png
+
+Besides creating an array from a sequence of elements, you can easily create an array filled with 0s:
+
+**Input:**
+
+::
+
+  np.zeros(2)
+
+**Output:**
+
+::
+
+  array([0., 0.])
+
+Or an array filled with 1s:
+
+**Input:**
+
+::
+
+  np.ones(2)
+
+**Output:**
+
+::
+
+  array([1., 1.])
+  
+Or even an empty array! The function ``empty`` creates an array whose initial content is random and depends on the state of the memory. 
+
+**Input:**
+
+::
+
+  # Create an empty array with 2 elements
+  np.empty(2)
+
+You can create an array with a range of elements:
+
+**Input:**
+
+::
+
+  np.arange(4)
+
+**Output:**
+
+::
+
+  array([0, 1, 2, 3])
+
+And even an array that contains a range of evenly spaced intervals. To do this, you will specify the **first number**, **last number**, and the **step size**.
+
+**Input:**
+
+::
+
+  np.arange(2,9,2)
+
+**Output:**
+
+::
+
+  array([2, 4, 6, 8])
+
+You can also use ``np.linspace()`` to create an array with values that are spaced linearly in a specified interval:
+
+**Input:**
+
+::
+
+  np.linspace(0,10,5)
+
+**Output:**
+
+::
+
+  array([ 0. ,  2.5,  5. ,  7.5, 10. ])
+
+**Specifying your data type**
+
+While the default data type is floating point (``float64``), you can explicitly specify which data type you want using ``dtype``.
+
+**Input:**
+
+::
+
+  array = np.ones(2, dtype=int)
+  array
+
+**Output:**
+
+::
+
+  array([1, 1])
+
+`Learn more about creating arrays here <https://docs.scipy.org/doc/numpy-1.17.0/user/quickstart.html#array-creation>`_.
+
+Adding, removing, and sorting elements
+--------------------------------------
+
+
+::
+
+  np.append()
+  np.delete() 
+  np.sort()
```

Comment:
Can we limit this section to just sort? Appending and deleting is super inefficient, and is never really used in well-written code.

Reply:
Perhaps use `np.concatenate` instead of `append`. And explain that deleting isn't really a thing; just use indexing instead to select the elements to keep.

Reply:
Got it!